src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -1047,7 +1047,7 @@ private void initializeConfig() {
         settingsPanel = new ModPanel();
         //int configStep = 40;
 
-        ModLabeledToggleButton allPacksModeBtn = new ModLabeledToggleButton(configStrings.TEXT[3], 350.0f, 600F, Settings.CREAM_COLOR, FontHelper.charDescFont, allPacksMode, settingsPanel, (label) -> {
+        FixedModLabeledToggleButton allPacksModeBtn = new FixedModLabeledToggleButton(configStrings.TEXT[3], 350.0f, 600F, Settings.CREAM_COLOR, FontHelper.charDescFont, allPacksMode, settingsPanel, (label) -> {
```

Comment:
The changes in this file are for the config menu -- a secondary issue that I only noticed because I looked at where else this button was used.